src/inference/src/dev/preprocessing/preprocessing.cpp

```diff
@@ -122,6 +122,17 @@ bool ov::pass::AddPreprocessing::run_on_model(const std::shared_ptr<ov::Model>&
         if (element_type != model->output(i).get_element_type()) {
             preproc.output(i).tensor().set_element_type(element_type);
         }
+
+        if (output_info->getLayout() != InferenceEngine::Layout::BLOCKED &&
+            output_info->getLayout() != InferenceEngine::Layout::SCALAR) {
+            std::stringstream stream;
+            stream << output_info->getLayout();
+            if (stream.str() == "NHWC") {
+                if (const_output.get_partial_shape().is_static() && const_output.get_shape().size() == 4)
+                    preproc.output(i).model().set_layout("NCHW");
```

Comment:
is model's layout is always NCHW?

Reply:
In plugin api 2.0, there is no layout info, so NCHW is the default layout align with input preproc?

Reply:
Can we have the same condition like for inputs:
```
if (const_output.get_partial_shape().is_static() && const_output.get_shape().size() == 4)
            preproc.output(i).model().set_layout("NCHW");
```
I think it should be enough...

Reply:
If I don't forget, it seems that some test case will fail if we don't set it, but I forget which test cases due to there are too many test failures at the beginning.